tools/hhsuite/.shed.yml

```diff
@@ -0,0 +1,5 @@
+description: HH-suite3 for sensitive sequence searching
+long_description: The HH-suite is an open-source software package for sensitive protein sequence searching based on the pairwise alignment of hidden Markov models (HMMs).
+homepage_url: https://github.com/soedinglab/hh-suite
+name: hhsuite
```

Comment:
If we want to add more tools to this `suite` later we should add a suite-based shed.yml. See the https://github.com/guerler/springsuite/blob/main/tools/springsuite/.shed.yml